modules/apps/client-extension/client-extension-type-item-selector-web/src/main/java/com/liferay/client/extension/type/item/selector/web/internal/item/selector/CETItemSelectorViewDescriptor.java

```diff
@@ -105,7 +106,17 @@ public boolean isShowBreadcrumb() {
 
 	private Predicate<CET> _getPredicate(String type) {
 		if (Objects.equals(
-				type, ClientExtensionEntryConstants.TYPE_GLOBAL_JS)) {
+				type, ClientExtensionEntryConstants.TYPE_GLOBAL_CSS)) {
+
+			return cet -> {
+				GlobalCSSCET globalJSCET = (GlobalCSSCET)cet;
```

Comment:
typo in variable name :mag:

Reply:
Ops, forgot to mention to just consider the last three commits @algobob

Reply:
oh cool

Reply:
@thiago-buarqque why is this included if it's not supposed to be reviewed?